client/state/a8c-for-agencies/agency/selectors.ts

```diff
@@ -34,3 +34,8 @@ export function hasAgency( state: A4AStore ): boolean {
 export function isAgencyClientUser( state: A4AStore ): boolean {
 	return state.a8cForAgencies.agencies.isAgencyClientUser;
 }
+
+export function hasAgencyCapability( state: A4AStore, capability: string ): boolean {
+	const agency = getActiveAgency( state );
+	return !! agency && agency?.user?.capabilities?.includes( capability );
```

Comment:
Can we do this instead? It will return undefined in some cases.

```suggestion
	return return agency?.user?.capabilities?.includes(capability) ?? false;
```

Reply:
Also, I think it's better to add the feature flag since this is directly affecting the production:

```
config.isEnabled( 'a4a-multi-user-support' ) 
```